einstein/forward.py

```diff
@@ -39,35 +41,44 @@ async def run_step(
     uids = get_random_uids(self, k=k, exclude=exclude or []).to(self.device)
 
     axons = [self.metagraph.axons[uid] for uid in uids]
+    bt.logging.info(f"[DEBUG] axons: {axons}")
```

Comment:
for debug log, it is bt.logging.debug (and to show this during the run, include the argument: ```--logging.debug``` in the command line